airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/util/Flattening.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.s3.util;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public enum Flattening {
```

Comment:
this looks like a copy-paste of `S3CsvFormatConfig:Flattening`. I think you should also refactor `S3CsvFormatConfig` to use this class to avoid code duplication.